rts/lra/client/src/main/java/io/narayana/lra/client/NarayanaLRAClient.java

```diff
@@ -305,7 +306,7 @@ public URI startLRA(URI parentLRA, String clientID, Long timeout, ChronoUnit uni
 
             response = client.target(base)
                 .path(START_PATH)
-                .queryParam(CLIENT_ID_PARAM_NAME, client)
+                .queryParam(CLIENT_ID_PARAM_NAME, clientID)
```

Comment:
This is a nice spot. I think the `clientID` would be better encoded (`URLEncoder.encode`).

Reply:
If you're going to encode it then use UTF-8:

> URLEncoder.encode(clientID, "UTF-8");

Reply:
it's done